src/service/viewer-impl.js

```diff
@@ -1005,7 +1005,7 @@ export class Viewer {
       return Promise.reject(getChannelError());
     }
     return this.messagingReadyPromise_.then(() => {
-      return this.sendMessageUnreliable_(eventType, data, awaitResponse);
+      return this.messageDeliverer_(eventType, data, awaitResponse);
```

Comment:
The naming is unimportant (this could be called `#postMessage` and `#reliablyPostMessage`). The benefit is the de-duplication.